pkg/test/resourcefixture/testdata/basic/alloydb/v1beta1/alloydbcluster/pscalloydbcluster/create.yaml

```diff
@@ -0,0 +1,30 @@
+# Copyright 2024 Google LLC
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#      http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+apiVersion: alloydb.cnrm.cloud.google.com/v1beta1
+kind: AlloyDBCluster
+metadata:
+  name: alloydbcluster-${uniqueId}
+spec:
+  displayName: test alloydb cluster
+  location: northamerica-northeast1
+  pscConfig:
+    pscEnabled: true
+  projectRef:
+    external: ${projectId}
+  initialUser:
+    user: postgres
+    password:
+      value: postgres
+  resourceID: alloydbcluster${uniqueId}
```

Comment:
nit: Add a newline at the end of the file.

Same comment applies to other yaml files.